runtime/picasso/build.rs

```diff
@@ -1,9 +1,10 @@
-use substrate_wasm_builder::WasmBuilder;
-
 fn main() {
-	WasmBuilder::new()
-		.with_current_project()
-		.export_heap_base()
-		.import_memory()
-		.build()
+	#[cfg(feature = "wasm-builder")]
+	{
+		substrate_wasm_builder::WasmBuilder::new()
+			.with_current_project()
+			.export_heap_base()
+			.import_memory()
+			.build()
+	}
```

Comment:
Rationale behind adding this feature? It's not harmful but I am curious :)

Reply:
to avoid building wasm when running native

Reply:
this speed ups build of simulator tests

Reply:
Hm, isn't this avoided by SKIP_WASM_BUILD?